gateway/datastructures/users.py

```diff
@@ -7,28 +8,46 @@ class UsernamePasswordForm(BaseModel):
 
 
 class UserForm(UsernamePasswordForm):
-    email: str = None
-    full_name: str = None
+    username: str
+    email: str
+    address: str
+    state_province: str
+    city: str
+    country: str
+    zip_code: str
+    hashed_password: str = None
     user_type: str
+    banned: int
+    profile_picture : str =  None 
 
 
 class UserUpdateForm(BaseModel):
-    username: str = None
-    email: str = None
-    full_name: str = None
-    user_type: str = None
+    username: str 
+    email: str 
+    address: str 
+    state_province: str 
+    city: str 
+    country: str 
+    zip_code: str 
+    hashed_password: str = None
+    user_type: str 
+    banned: int = None
+    profile_picture : str =  None 
 
 
 class LoginResponse(BaseModel):
     access_token: str
     token_type: str
 
 
-class UserResponse(BaseModel):
-    id: int
-    username: str
+class UserResponse(BaseModel):        
+    username: str = None
     email: str = None
-    full_name: str = None
```

Comment:
I think full name should be part of the user calls, or at least first_name and last_name, front end expects this information